tasks/configure.yml

```diff
@@ -21,6 +21,54 @@
   tags:
     - rstudio-workbench-configure-session
 
+- name: configure | create (load-balancer) configuration file
+  template:
+    src: etc/rstudio/load-balancer.j2
+    dest: /etc/rstudio/load-balancer
+    owner: root
+    group: root
+    mode: 0644
+  notify: restart rstudio-workbench
+  when: rstudio_workbench_load_balancer_config_override is defined and rstudio_workbench_load_balancer_config_override|length
+  tags:
+    - rstudio-workbench-configure-load-balancer
+
+- name: configure | create (openid) configuration file
+  template:
+    src: etc/rstudio/openid-client-secret.j2
+    dest: /etc/rstudio/openid-client-secret
+    owner: root
+    group: root
+    mode: 0600
+  notify: restart rstudio-workbench
+  when: (rstudio_workbench_openid_client_id is defined and rstudio_workbench_openid_client_id|length) and (rstudio_workbench_openid_client_secret is defined and rstudio_workbench_openid_client_secret|length)
```

Comment:
`is defined` is [sufficient](https://docs.ansible.com/ansible/latest/user_guide/playbooks_conditionals.html#conditionals-based-on-variables) for `when:` to check if variable is defined and not empty.